spec/pluralrules.html

```diff
@@ -70,7 +68,7 @@
            1. Let _fv_ be the substring of _s_ from position _dp_, exclusive.
            1. Let _f_ be ? ToNumber(_fv_).
            1. Let _v_ be ? ToLength(? Get(_fv_, *"length"*)).
-         1. Let _i_ be ? ToNumber(_iv_).
+         1. Let _i_ be ? abs(? ToNumber(_iv_)).
```

Comment:
The `abs()` is needed, to account for the case where _iv_ encodes a negative number. Note that this PR also removes the `abs()` operation that was on line 61, and without such we may end up with a negative number for `[[IntegerDigits]]`.